test/e2e/proxy_test.go

```diff
@@ -30,6 +30,18 @@ import (
 	"antrea.io/antrea/pkg/features"
 )
 
+type agnhostEndpoint string
+
+var (
+	emptyEp           agnhostEndpoint = ""
+	clientIPEp        agnhostEndpoint = "clientip"
+	hostNetworkPodEP  agnhostEndpoint = "shell?cmd=echo+$hostnetwork"
+	hostNetworkNodeEP agnhostEndpoint = "shell?cmd=echo+\\$hostnetwork"
```

Comment:
maybe add a quick comment on why these 2 are different?

Reply:
Added a simple comment. I didn't find an official explanation about why escape is needed on Node and there're few docs/discussions online about agnhost netexec.